clients/web/src/lib/connection/monitor.ts

```diff
@@ -42,11 +63,24 @@ export type MonitorData = {
   connectionStatus: HealthStatus;
 };
 
+export function initialDurations() {
+  return {
+    start: undefined,
+    end: Date.now() * 1e6,
```

Comment:
Maybe use `Infinity` to show there's no end yet. Yes, that's a valid number in JS/TS.